### ISSUE TITLE
Still top result in Google. Still not the best choice for users

### ISSUE THREAD

**Issue by andybak** [sciyoshi/django-dbsettings#18]
PLEASE fix this. https://github.com/zlorf/django-dbsettings is a much better option for anyone adopting this project but there is no indication in the README that this fork is moribund. Even I have to keep going to PyPi to remind myself of the correct fork to use. Can you either redirect to zlorf or at least add a big message to the top of your readme?

**Comment by zlorf** [comment]
Or I should merge back.

**Comment by andybak** [comment]
Cool. That would work too but would need to be an ongoing effort (unless you're prepared to give others commit rights?)

**Comment by andybak** [comment]
Another thing to consider is the official PyPi version points to zlorf and I assume he has ownership of that PyPi id

**Comment by zlorf** [comment]
I think that the note in the README is clear enough; alternatively @sciyoshi can also put a note just below the repo name, something like here: https://github.com/aanand/git-up

But eventually I'll make a backport of my code since I have a write access to this repo; I just need to get a little of time.

**Comment by sciyoshi** [comment]
Thanks @zlorf - I like that idea better. I've removed the commit with the README.md changes and I've added the note instead to the description. I've also merged in the latest changes from your repository into here. At this point there's only two small differences that are in this version: https://github.com/zlorf/django-dbsettings/compare/master...sciyoshi:master

Again, I'd be happy to move this to a Django-sponsored project if there's momentum around that.
